target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -424,7 +424,8 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
 		;;
-	tl-wr842n-v2)
+	tl-wr842n-v2|\
+	tl-wr942n-v1)
 		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
```

Comment:
> This is meaningless in case of production u-boot installed as RX GPIO is already assigned as OUTPUT by it. This will work with beta u-boots only. I think we should check GPIO 14 and 15 asignments on board initialisation time on kernel side and reassign it to LEDS if it is not.

Or make it static (always set GPIO14/15 as regular GPIO), without checking what U-Boot configured earlier.

In future, when we switch ar71xx target to dts, there will/should be easy way to produce two images with different dtb appended to the kernel - one with regular/default GPIO definition (for USB LEDs) and one with UART support on these GPIOs. And that wouldn't require to have any _hacks_ inside board specific code.